controllers/apply-next/simple/fields.js

```diff
@@ -278,16 +278,11 @@ const allFields = {
             if (!isArray(value)) {
                 return value;
             } else {
-                const total = value.reduce((acc, cur) => {
-                    return acc + (parseInt(cur.cost || 0) || 0);
-                }, 0);
-                let str = '<ul>';
-                value.forEach(line => {
-                    str += `<li>${line.item} &ndash; £${line.cost}</li>`;
-                });
-                str += '</ul>';
-                str += `<p><strong>Total</strong>: £${total}`;
-                return str;
+                const total = sumBy(value, item => parseInt(item.cost || 0));
```

Comment:
For the time being I've restricted this to a text only representation so we can keep some of the safety in views. Will revisit this as I do think it's useful to have _some_ way of rendering markup for some fields.